event.graphqls

```diff
@@ -0,0 +1,64 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+scalar Long
+
+type Event {
+    uuid: String!
+    source: Source!
+    name: String!
+    type: Type!
+    message: String
+    parameters: String
+    startTime: Long!
+    endTime: Long
+}
+
+type Source {
+    service: String
+    serviceInstance: String
+    endpoint: String
+}
+
+input SourceInput {
+    service: String
+    serviceInstance: String
+    endpoint: String
+}
+
+enum Type {
+    Normal
+    Error
+}
+
+type Events {
+    events: [Event!]!
+    total: Int!
+}
+
+input EventQueryCondition {
+    uuid: String
+    source: SourceInput
+    name: String
+    type: Type
+    startTime: Duration
+    endTime: Duration
```

Comment:
This seems not correct, take a look at here, https://github.com/apache/skywalking-query-protocol/blob/master/trace.graphqls#L42